tdwg/tdwg-theme
# Design homepage

Design jumbotron as:

* Jumbotron with title (`page.title`) and short description of TDWG (`page.summary`). Background image is currently chosen because it fits well with TDWG blue
* Introduction to TDWG (`page.content`: currently from constitution) and alert box (`page.alert` properties) to highlight current events (e.g. conference)
* 3 most popular standards (`page.tags`)
* Latest 2 news items
* Subscription to newsletter link

![homepage](https://user-images.githubusercontent.com/600993/36105248-69760986-1014-11e8-913a-98ff2a639d33.png)

Should scale well on all screens.

Further:
* Make use of https://github.com/peterdesmet/pelican-cover-image to check for cover image or using a default
* Use `.badge` for tags
* Update to bootstrap 4.0
* Update font to Helvetica Neue
* Use padding bottom for cover images for better scaling (thanks @MortenHofft)
* Make use of GBIF Thumbor service at https://api.gbif.org/v1/image (thanks @MortenHofft)
* Update some metadata tags
